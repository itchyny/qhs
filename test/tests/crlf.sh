qhs -H -O "SELECT * FROM crlf.csv"
