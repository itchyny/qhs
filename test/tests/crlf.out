foo bar baz
a0 1 a2
b0 3 b2
c0 c2 
